src/main/java/org/datadog/jmxfetch/Instance.java

```diff
@@ -257,6 +266,19 @@ public Instance(
         } else {
             log.info("collect_default_jvm_metrics is false - not collecting default JVM metrics");
         }
+        //create bean to track data of this instance
+        JMXbean = beanManager.createJMXBean(this);
```

Comment:
This uses a static instance of the `BeanManager` class to create each telemetry bean, but I'm not sure why we need the static instance and the "factory" like aspect of this.

Is there a reason why we cannot simply create a new instance of the "telemetry bean" from `Instance` directly and set it as an instance property?

This would remove the need for `BeanManager` entirely and simplify the code structure here